pyaedt/modules/PostProcessor.py

```diff
@@ -547,18 +549,77 @@ def eigenmode(self, expressions=None, setup_name=None):
         return
 
     @pyaedt_function_handler()
-    def eye_diagram(self, expressions=None, setup_name=None):
+    def statistical_eye_contour(self, expressions=None, setup_name=None, quantity_type=3):
+        """Create a Standard Statistical AMI Contour plot.
```

Comment:
```suggestion
        """Create a standard statistical AMI contour plot.
```
Will users know what AMI  means?